code/modules/mob/living/simple_animal/hostile/giant_spider.dm

```diff
@@ -88,6 +90,13 @@
 		return TRUE
 	return ..()
 
+/mob/living/simple_animal/hostile/giant_spider/expose_reagents(list/reagents, datum/reagents/source, methods=TOUCH, volume_modifier=1, show_message=TRUE)
```

Comment:
Can't we put this in the same block of code they use to apply the brute damage? In the reagent's ``expose_mob()``?

Reply:
We can, though I was personally a fan of keeping it here since this interaction only applies to spiders and having it in their file keeps an important interaction such as this within eyeshot.